### PR TITLE
[CGPROD-2768] shop confirm screen - part 1

### PR DESCRIPTION
**Pull request by BeeGodwin**
- Adds layout for the confirm screen
- Resize and visibility functions reused from the shop menu
- Refactor menu-buttons.js for reuse
- Placeholder images etc for items
- Not hooked up to anything yet!

### CODE REVIEW

File: src/components/shop/confirm.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+export const createConfirm = (scene, config, bounds) => {
+    const { buttonsRight } = config.menu;
+    const { styleDefaults } = config;
+
+    const confirmContainer = scene.add.container();
+    const innerBounds = getOffsetBounds(bounds, getInnerRectBounds(bounds, buttonsRight));
+    const yOffset = bounds.height / 2 + bounds.y;
+
+    confirmContainer.buttons = createConfirmButtons(scene, innerBounds, config, yOffset); // hard coded value nono
```
Comment (FostUK) [inline src/components/shop/confirm.js:19]: comment?

File: src/components/shop/confirm.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    confirmContainer.add([
+        createRect(scene, getHalfRectBounds(bounds, !buttonsRight), 0xff0000),
+        createRect(scene, getHalfRectBounds(bounds, buttonsRight), 0xff00ff),
+        createRect(scene, innerBounds, 0x0000ff),
+        scene.add.text(innerBounds.x, promptY(bounds), config.confirm.prompts.buy, styleDefaults).setOrigin(0.5),
+        scene.add.text(innerBounds.x + 20, currencyY(bounds), "1000", styleDefaults).setOrigin(0.5),
+        scene.add.image(innerBounds.x - 20, currencyY(bounds), "shop.balanceIcon"),
```
Comment (FostUK) [inline src/components/shop/confirm.js:27]: Possibly using "shop.balanceIcon" might be a bad idea - e.g: there could be multiple shops and they'll be named whatever they are called in `main,js`.

If you need to merge any of this now though just make a note we need to cover these options before release.

**Review by FostUK — APPROVED** [review]
Looks good - just some really minor comments :) 

🌭

File: src/components/shop/confirm.js (hunk 2 above)
Comment (BeeGodwin) [reply src/components/shop/confirm.js:27]: Yeah, this should be pulling a value from config- fixed.

File: src/components/shop/confirm.js (hunk 1 above)
Comment (BeeGodwin) [reply src/components/shop/confirm.js:19]: Good spot! removed.

**Pull request merged**